data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+	on offer
+		conversation
+			`You see an Arach coming toward you as you enter the spaceport, and recognize him as Consul Plortup, the one who you helped deploy drones on a Saryd world.`
+			`	"Ask for your assistance, once again, I must, Captain <last>," he says.`
+			`	You have no time to ask what's going on, or refuse, as he has you brought to a separate room where a Saryd, Kimek, and Arach are waiting.`
+			`	"Brewing on <destination>, another riot is. Your continued support, we hope you'll once more provide, Captain."`
+			choice
+				`	"Alright, what's the plan?"`
+					goto plan
+				`	"I'm not really comfortable in helping with these. Can you let me go?"`
+			`	He perks his head up a bit, then mumbles something in his own language, momentarily turning off the translator. He gives in, and has someone bring you back to your ship.`
+				decline
+			label plan
+			`	"Granted access to our civilian ships, you were. Three new Kimek Spires, you have bought. Many passengers to <destination>, in them you are carrying," he says.`
+			`	The Saryd of the trio of captains then speaks up, "Pose as civilian ships, we must. Accompany your ship, we will."`
+			`	"Armed and ready, the soldiers are. Follow your lead once you depart, we shall," the Kimek adds.`
+			`	Plortup wishes you all good luck, and you follow the trio of captains back to your ship, seeing that the Spires are parked close to it.`
+				accept
+	npc accompany save
+		personality escort heroic opportunistic
+		government "Coalition"
+		ship "Kimek Spire" "Kiry Paichee"
+		ship "Kimek Spire" "Achipa Kyrri"
+		ship "Kimek Spire" "Pakee Pikary"
+	on visit
+		dialog `You've reached <planet>, but the Heliarch transports haven't arrived yet! Best depart and wait for them to get here.`
+
+mission "Heliarch Containment 4-B"
+	name "Undercover: Payment"
+	description "Collect your payment for escorting the Heliarch transports."
+	landing
+	to offer
+		has "Heliarch Containment 4-A: done"
+		not "Resistance: Questions: done"
+	source "Remote Blue"
+	destination "Station Cian"
+	on offer
+		conversation
+			`The hundreds of Heliarch agents leave the three transports, rushing in groups to separate parts of the city. They march in unison, weapons in hand.`
+			`	You're informed by one of the local Heliarchs nearby that you should now head to <destination> to collect your payment of <payment>.`
+				accept
+	on complete
+		payment 495000
+		conversation
+			`You land in the station to find that <payment> have been transferred to you. Along with the credits comes a transmission from Plortup. "For my absence, forgive me you must, Captain. On Fourth Shadow, broken out another riot has, so organizing some ships from within the Ring of Power, I am."`
+			`	He goes on to explain that the riot was quite a large one, and much of the fleet stationed here was sent there to help contain it. What remained is helping with the Remote Blue riot, so the station is fairly devoid of a military presence now.`
```

Comment:
There's a lot of riots by this point, I don't know if it's addressed in the resistance side yet, but this would imply *widespread* unrest among the population with easily identifiable cause.

Reply:
Heliarchs wouldn't use the word riot, will change to attack or something like that so that it doesn't imply it's civilians.